eventstream.go

```diff
@@ -200,6 +201,6 @@ func NewClient(prefix, stream string, brokers []string, config ...*BrokerConfig)
 
 // Client is an interface for event stream functionality
 type Client interface {
-	Publish(publishBuilder *PublishBuilder) error
-	Register(subscribeBuilder *SubscribeBuilder) error
+	Publish(publishBuilder *PublishBuilder)
+	Register(subscribeBuilder *SubscribeBuilder)
```

Comment:
how would the publish and register caller will know and act if there's any error?

Reply:
I think publish / register event shouldn't blocking and doesn't overload caller to handle the error.

Reply:
I agree that they should not block the caller, but the caller needs to know if the event is not sent

Reply:
We can know from the logs, I think we can set default log mode to error level

Reply:
that's debugging, the service who calls this code needs to know if the call is successful or not. It's minimum basic resiliency needed for microservices, knowing when there's an error so that the service can act on it

Reply:
previously, we only return error when validation is failed
do you think we need to return error when publish an event also? if we do that, it will increase request time of client / service because sometimes we retrying to publish event with exponential backoff.

Reply:
it can still be async to return the result

Reply:
okay, I will rolled back like previous version
will catch error for validation. thanks